### ISSUE TITLE
Pre-Updater Checker

### ISSUE THREAD

**Issue by machadoug** [elkuku/EasyCreator#48]
Will EasyCreator implement Joomla's 3.9 Pre-Updater Checker for third party extensions?
https://github.com/joomla/joomla-cms/pull/17742

**Comment by elkuku** [comment]
Yes I think this should definitely be implemented. Not sure when TBH... 
But helping hands are always welcome 😉
